azure-specialized-workloads/oracle/recommendations.yaml

```diff
@@ -0,0 +1,201 @@
+  potentialBenefits: Ensure business continuity in case of regional failure.
+  pgVerified: false
+  publishedToLearn: false
+  automationAvailable: no
+  tags: null
+  learnMoreLink:
+    - name: Business continuity and disaster recovery considerations for Oracle Database@Azure
+      url: "https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/oracle-iaas/oracle-disaster-recovery-oracle-database-azure"
+    - name: Learn about Oracle Maximum Availability Architecture for Oracle Database@Azure
+      url: "https://docs.oracle.com/en/solutions/oracle-maa-db-at-azure/#GUID-7723E2B1-9588-40BC-88BE-44637B1AF0D9"
+    - name: Oracle Database@Azure Evaluations by Oracle MAA
+      url: "https://docs.oracle.com/en/database/oracle/oracle-database/21/haovw/db-azure1.html#GUID-91572193-DF8E-4D7A-AF65-7A803B89E840"
+
+
+- description: Implement a strategy for resiliency that is tailored to your Oracle workload requirements
+  aprlGuid: e2750bd4-d12c-409a-bb56-49238d9a8013
+  recommendationTypeId: null
+  recommendationControl: Business Continuity
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/oracle
+  recommendationMetadataState: Active
+  longDescription:
+    Define a BCDR strategy based on the workload's RPO/RTO requirements.
+    Oracle has set of best practices and reference architectures tailored to meet various workload RPO/RTO requirements. This framework is known as the Oracle Maximum Availability Architecture (MAA).
+    By default, The Oracle Database@Azure solution follows a Silver level reference architecture. The primary database residing in ExaDB-D provides high availability, data protection, elasticity, and scalability benefits, however it is vulnerable to any AZ failure.
+    Gold service level reference architectures with Oracle Database@Azure is recommended. It could be within the same Azure Region or across one or more Azure Regions.
+  potentialBenefits: The cloud MAA architecture achieves data protection and DR.
+  pgVerified: false
+  publishedToLearn: false
+  automationAvailable: no
+  tags: null
+  learnMoreLink:
+    - name: Business continuity and disaster recovery considerations for Oracle Database@Azure
+      url: "https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/oracle-iaas/oracle-disaster-recovery-oracle-database-azure"
+    - name: Learn about Oracle Maximum Availability Architecture for Oracle Database@Azure
+      url: "https://docs.oracle.com/en/solutions/oracle-maa-db-at-azure/#GUID-7723E2B1-9588-40BC-88BE-44637B1AF0D9"
+    - name: Oracle Database@Azure Evaluations by Oracle MAA
+      url: "https://docs.oracle.com/en/database/oracle/oracle-database/21/haovw/db-azure1.html#GUID-91572193-DF8E-4D7A-AF65-7A803B89E840"
+
+- description: Implement a backup and restore strategy for Oracle databases to meet your workload requirements
```

Comment:
This appears to be more architectural guidance than a configuration for the Oracle resource that can be configured to improve the resiliency of the Oracle resource deployment. As such, wouldn't this better be addressed through WAF guidance?

Reply:
The recommendation is agreed by relevant teams and as such we would like to keep it. Please reach on Teams if you need further discussion with the relevant team experts. thanks

Reply:
Agree with @ehaslett this reads as a generic best practice rather than an actionable reliability recommendation. We also already have WAF recommendation RE:09 that covers the same topic.

Reply:
@basimolimajeed , due to the APRL requirements @ehaslett mentions [above](https://github.com/azure/azure-proactive-resiliency-library-v2/pull/463#discussion_r1819500407), this should be moved to WAF.